src/Wrapper.tsx

```diff
@@ -171,7 +170,7 @@ export default function Wrapper({ title, page, children }: { title: any; page: s
       <Banner />
       <Box sx={{ display: 'flex' }}>
         <CssBaseline />
-        {!isUiConfigLoading && uiConfig!.banner.enable && <Box sx={{ mt: 20 }} />}
+        {!isUiConfigLoading && uiConfig?.banner.enable && <Box sx={{ mt: 20 }} />}
```

Comment:
Replacing this with `{uiConfig && uiConfig.banner.enable...` prevents TypeScript getting confused and requiring a bunch of `?` everywhere.

Reply:
Done